libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -702,15 +702,11 @@ bool ContractCompiler::visit(FunctionDefinition const& _function)
 bool ContractCompiler::visit(InlineAssembly const& _inlineAssembly)
 {
 	unsigned startStackHeight = m_context.stackHeight();
-	yul::ExternalIdentifierAccess identifierAccess;
-	identifierAccess.resolve = [&](yul::Identifier const& _identifier, yul::IdentifierContext, bool)
```

Comment:
Is this not done anymore?

Reply:
That was the trigger for this refactor, this function is unused in the code generator.